client/app/cloud/project/compute/infrastructure/privateNetwork/cloud-project-compute-infrastructure-privateNetwork-list.directive.js

```diff
@@ -81,6 +82,14 @@ class PrivateNetworkListCtrl {
     this.User.v6().get().$promise.then((user) => {
       this.orderUrl = _.get(this.URLS.website_order, `vrack.${user.ovhSubsidiary}`);
     });
+    this.VrackService.listOperations(this.$stateParams.projectId)
+      .then((result) => {
+        const status = _.filter(result, f => f.status !== 'completed');
+        if (status.length > 0) {
+          this.loaders.vrack.link = true;
+          this.pollOperationStatus(status[0].id);
+        }
+      });
```

Comment:
Could those promises be chained ?

Reply:
They are not dependent on each other. Will load page faster if they are executed as they are

Reply:
Are the performances really an issue ? Because it would make code easier to read if the Promises were chained :)